test/harness/services/gossip/adapter/tampering_transport_test.go

```diff
@@ -21,7 +22,7 @@ func newTamperingHarness() *tamperingHarness {
 	senderKey := "sender"
 	listenerKey := "listener"
 	listener := &mockListener{}
-	transport := NewTamperingTransport()
+	transport := NewTamperingTransport(log.GetLogger(log.Service("transport")))
```

Comment:
`transport` isn't really a service.. The convention I used so far in the system was `log.String("adapter", "gossip")`. Search the project for `log.String("adapter",` - we have a few of those already (in all the adapters that have logs).

Maybe we should add `log.Adapter("gossip")` which is a parallel to `log.Service("gossip")`

Reply:
yes